pkg/api/api.go

```diff
@@ -195,6 +198,12 @@ func (s *Service) SetSwarmAddress(addr *swarm.Address) {
 	}
 }
 
+func (s *Service) SetRedistributionAgent(redistributionAgent *storageincentives.Agent) {
```

Comment:
this is not used anywhere

Reply:
I missed it in node.go file. Let me add it.